release-notes/3.0/3.0-supported-os.md

```diff
@@ -14,8 +14,8 @@ OS                            | Version                       | Architectures  |
 ------------------------------|-------------------------------|----------------|-----
 Windows Client                | 7 SP1+, 8.1                   | x64, x86       |
 Windows 10 Client             | Version 1607+                 | x64, x86       |
-Nano Server                   | Version 1803+                 | x64, ARM32     |
-Windows Server                | 2012 R2+                      | x64, x86       |
```

Comment:
It occurred to me after creating the PR that these could be referring to WOW64 specifically. 🤔 If that's the case, perhaps a footnote clarifying this could be worthwhile?